docs/tut-making-jettycat.md

```diff
@@ -389,17 +396,17 @@ Here we compare a copy's horizontal coordinate to a camera's left side. We also
 
 Let's add a template for star bonuses that will increment score when collected. We will do the following:
 
-1. Set up a score variable in our room's "On Create" code.
+1. Set up a score variable in our room's "Creation" code.
 2. Create a new template for star bonuses.
-3. Add a bit of logic to the star's "On Step" event that will destroy the star when collided with the cat.
+3. Add a bit of logic to the star's "Frame start" event that will destroy the star when colliding with the cat.
 4. Create a new room and a template for it to display a score counter.
 5. Put this new room into the main one.
 
-Now, open the `InGame` room's events and add a line `this.score = 0;`. This will create a variable we will be able to edit and read in any other copy.
+Now, open the `InGame` room's events and add a line `this.score = 0;` to the Room start event. This will create a variable we will be able to edit and read in any other copy.
 
 Create a new template, and call it a `Star`. Set its texture.
 
-In its On Step code, put this script:
+In its Frame start code, put this script:
 
 ```js
```

Comment:
I think use the collision event but note the alternative way with `ct.place.meet` in a tip after the main instruction.

Tips can be created with
```
::: tip
these
:::
```